style.tex

```diff
@@ -59,6 +59,7 @@
 \lstset{basicstyle=\ttfamily,breaklines=true, keepspaces=true}
 \lstdefinestyle{prompt}{
 % language=bash,
+  basicstyle=\footnotesize\ttfamily,
```

Comment:
addition of these basicstyle seem to break the minipage setup declared in `\lstnewenvironment{prompt}`
prompts and flattext are now again split over multiple pages.